spot-client/src/common/remote-control/P2PSignalingBase.js

```diff
@@ -55,6 +55,21 @@ export default class P2PSignalingBase extends Emitter {
         return this._peerConnections.get(remoteAddress);
     }
 
+    /**
+     * Finds if any connections are currently alive.
+     *
+     * @returns {boolean}
+     */
+    hasActiveConnection() {
+        for (const peerConnection of this._peerConnections.values()) {
```

Comment:
Is it possible to use `find` on `.values()` ?

Reply:
Not that I know of because that's an iterator, but I've made a change to turn the peerConnections map into an array so .find can be called on it.